libpretixsync/src/main/java/eu/pretix/libpretixsync/db/Migrations.java

```diff
@@ -15,7 +15,7 @@
 
 public class Migrations {
     private static EntityModel model = Models.DEFAULT;
-    public static int CURRENT_VERSION = 105;
+    public static int CURRENT_VERSION = 106;
```

Comment:
An annoying detail that you might not be aware of: we currently have to define the DB version in two places, since SQLDelight has a different mechanism.
It deduces the DB version from its migration files. Since they are only placeholders for now, we can simply rename `sqldelight/migrations/104.sqm` to `105.sqm` (it's named after the version you're migrating _from_). Or you could create another empty `105.sqm` next to the existing migration